[Docs] Add deprecation notices for Azure OpenAI, Amazon Bedrock, and Google Gemini connectors (#279551)
elastic/kibana: docs/reference/connectors-kibana/bedrock-action-type.md

Reviewer: ```suggestion
stack: deprecated 9.6
serverless: deprecated
```

Author: oh yeah, I guess we could add this detail early. It should appear as "planned" in the HTML rendering anyways.

mapped_pages:
  - https://www.elastic.co/guide/en/kibana/current/bedrock-action-type.html
applies_to:

>   stack: deprecated 9.6

Reviewer: ```suggestion
  stack: deprecated 9.5
```

  - https://www.elastic.co/guide/en/kibana/current/bedrock-action-type.html
applies_to:

>   stack: all

Reviewer: ```suggestion
```

stack: deprecated 9.6

> serverless: deprecated

Reviewer: ```suggestion
  stack: deprecated 9.6
  serverless: deprecated
```

  - https://www.elastic.co/guide/en/kibana/current/bedrock-action-type.html
applies_to:
  stack: all
stack: deprecated 9.6

> serverless: deprecated

Author: fixed

The {{bedrock}} connector uses [axios](https://github.com/axios/axios) to send a POST request to {{bedrock}}.

::::{important}

> :applies_to: {"stack": "deprecated 9.5, removed 9.6", "serverless": "removed"}

Reviewer: ```suggestion
:applies_to: {"stack": "deprecated 9.5", "serverless": "deprecated"}
```
If those already in place continue to work I would just call it deprecated. For me removed = no longer works

Author: Unless I'm misunderstanding our guidance on using the `removed` value, I think the current configuration (`:applies_to: {"stack": "deprecated 9.5, removed 9.6", "serverless": "removed"}`) is correct.

<img width="709" height="325" alt="Screenshot 2026-07-21 at 12 19 34 PM" src="https://github.com/user-attachments/assets/487e5945-d386-42f2-88e1-b24682596ab2" />

These doc updates also include a line that explicitly states existing connectors still work, but are marked as deprecated in the UI.

Author: Approving @florent-leborgne's edit since the connectors are hidden in serverless/9.6 and will eventually be removed.

The {{bedrock}} connector is deprecated. Existing {{bedrock}} connectors and rule actions that use them continue to work but display deprecation indicators. Follow the guidance that matches your version and deployment type:

- {applies_to}`serverless: removed` {applies_to}`stack: removed 9.6` You cannot create new LLM connectors through the standard connector UI.

> - {applies_to}`stack: deprecated 9.5` For new AI integrations, use {{es}} {{infer}} endpoints instead of LLM connectors. Plan migration of existing LLM connectors and any rule actions that reference them before future removal.

Reviewer: ```suggestion
This connector is deprecated. If it is in use, it continues to work, along with any rule actions that rely on it. 

For new AI integrations, use {{es}} {{infer}} endpoints instead of LLM connectors. Plan migration of existing LLM connectors and any rule actions that reference them before future removal.
```
Wondering if we can say something like this for simplification, and skip the "removed" bullet. Or maybe add something untagged like "It is progressively being removed from the available options when creating new LLM connectors."

Author: Yeah, I like this simplified version.

Reviewer: wdyt of the second part of the suggestion: "It is progressively being removed from the available options when creating new LLM connectors."?

(Also I only commented on this file but I guess we'd do the same for all of them?)

Author: A combo of that and your revision would work, for example:

This connector is deprecated and is being progressively removed from the create connector UI. Existing connectors and their rule actions continue to work.

For new AI integrations, use {{es}} {{infer}} endpoints. Migrate existing LLM connectors and related rule actions before the future removal.

Reviewer: nice
